docs/settings/alert-action-settings.asciidoc

```diff
@@ -179,3 +179,10 @@ For example, `20m`, `24h`, `7d`, `1w`. Default: `60s`.
 
 `xpack.alerting.maxEphemeralActionsPerAlert`::
 Sets the number of actions that will be executed ephemerally. To use this, enable ephemeral tasks in task manager first with <<task-manager-settings,`xpack.task_manager.ephemeral_tasks.enabled`>>
+
+`xpack.alerting.defaultRuleTaskTimeout`::
+Specifies the default timeout for the all rule types tasks. Task timeout could be overridden by the rule type producer during registration. The time is formatted as:
```

Comment:
I don't think we need to specify here that the task timeout can be overridden since these are user docs and users can't do anything about that. I do think we should say what the default value is.